### PR TITLE
setting the correct TITLE (WIDTHxHEIGHT) based on the generator answers

### PR DESCRIPTION
**Pull request by caos2k**

### CODE REVIEW

**Comment by mientjan** [comment]
Can you also do this for netflix?

**Comment by caos2k** [comment]
sure, I was actually starting doing it

File: generators/createBanner/templates/doubleclick/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <title></title>
+  <title><%= banner_width %>x<%= banner_height %></title>
```
Comment (mientjan) [inline generators/createBanner/templates/doubleclick/index.html:7]: spaces are wrong, check this for the next time. Not importent now.

File: generators/createBanner/index.js [hunk 2]
```diff
@@ -103,9 +117,17 @@ module.exports = class extends Generator {
           json,
         );
 
-        mkdirp(this.destinationPath(path.join(this.result.outputPath, 'img')), err => {
-          if (err) console.error(err);
-        });
+        // removing this since the img folder has to be there for the loader, therefore I'' not creating the folder but just copyTpl
```
Comment (mientjan) [inline generators/createBanner/index.js:120]: Just remove the code.

File: generators/createBanner/templates/doubleclick/css/style.css [hunk 3]
```diff
@@ -1,3 +1,56 @@
...
+
+body {
+  background-color: #FFF;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.hide {
```
Comment (mientjan) [inline generators/createBanner/templates/doubleclick/css/style.css:25]: This is gone. Not used anymore are we still going to use this?

File: generators/createBanner/templates/doubleclick/css/style.css [hunk 4]
```diff
@@ -1,3 +1,56 @@
...
+}
+
+.bannerClick {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+  background: url('data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7');
```
Comment (mientjan) [inline generators/createBanner/templates/doubleclick/css/style.css:55]: can we swap this with a transparent pixel image?

**Pull request merged**